ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyOptimizer.java

```diff
@@ -437,7 +437,7 @@ private void transitionLayoutOnlyViewToNativeView(
 
   private static boolean isLayoutOnlyAndCollapsable(@Nullable ReactStylesDiffMap props) {
     if (props == null) {
-      return true;
+      return false;
```

Comment:
Yea I was going to comment on this, If I don't do this native view counts doesn't match sticky indices. I think it is because of `null` react elements. Any other idea other than doing this?